README.md

```diff
@@ -61,15 +62,15 @@ We provide a sample [docker-compose.yml](./proxy/ops/docker-compose.yml) file fo
 Once Docker compose is installed, you can test your specific configuration by running Docker compose interactively with:
 
 ```bash
-docker compose -f /path/to/this/repo/docker-compose.yml up
+docker-compose -f /path/to/this/repo/docker-compose.yml up
```

Comment:
Newer versions of the docker repository contain `docker-compose-plugin` where you can run `docker compose` as a plugin with docker engine. So this change is not required.